package.json

```diff
@@ -62,7 +63,8 @@
     "start": "npm-run-all -p watch-css start-js",
     "build-js": "react-scripts build",
     "build": "npm-run-all build-css build-js",
-    "test": "react-scripts test --env=node",
+    "test": "react-scripts test",
```

Comment:
Redundant parameter, as the `react-scripts` already have `env=node` as a default.